addons/l10n_pe/data/template/account.tax-pe.csv

```diff
@@ -1,61 +1,74 @@
-"","","","","","","","","","","","","tax","invoice","chart40111","",""
-"","","","","","","","","","","","","base","refund","","",""
-"","","","","","","","","","","","","tax","refund","chart40111","",""
-"purchase_tax_igv_18g_ng","18% Levied and Not Taxed","","IGV","1000","S","18.0","purchase","1","1","tax_group_igv_g_ng","True","base","invoice","","","18% Gravadas y No Gravada"
-"","","","","","","","","","","","","tax","invoice","chart40117","",""
-"","","","","","","","","","","","","base","refund","","",""
-"","","","","","","","","","","","","tax","refund","chart40117","",""
-"purchase_tax_igv_18_ng","18% Not Levied","","IGV NG","1000","S","18.0","purchase","1","1","tax_group_igv_ng","True","base","invoice","","","18% No Gravadas"
-"","","","","","","","","","","","","tax","invoice","chart40116","",""
-"","","","","","","","","","","","","base","refund","","",""
-"","","","","","","","","","","","","tax","refund","chart40116","",""
-"purchase_tax_exp_0","0% EXP","","EXP","9995","S","0.0","purchase","1","1","tax_group_exp","True","base","invoice","","","0% EXP"
-"","","","","","","","","","","","","tax","invoice","chart40115","",""
-"","","","","","","","","","","","","base","refund","","",""
-"","","","","","","","","","","","","tax","refund","chart40115","",""
-"purchase_tax_exo","0% Exo","","EXO","9997","E","0.0","purchase","1","","tax_group_exo","","base","invoice","","","0% Exonerado"
-"","","","","","","","","","","","","tax","invoice","chart40111","",""
-"","","","","","","","","","","","","base","refund","","",""
-"","","","","","","","","","","","","tax","refund","chart40111","",""
-"purchase_tax_ina","0% Una","","INA","9998","Z","0.0","purchase","1","","tax_group_ina","","base","invoice","","","0% Inafectado"
-"","","","","","","","","","","","","tax","invoice","chart40111","",""
-"","","","","","","","","","","","","base","refund","","",""
-"","","","","","","","","","","","","tax","refund","chart40111","",""
-"purchase_tax_gra","0%","","GRA","9996","E","0.0","purchase","1","","tax_group_gra","","base","invoice","","","0% Gratis"
-"","","","","","","","","","","","","tax","invoice","chart40111","",""
-"","","","","","","","","","","","","base","refund","","",""
-"","","","","","","","","","","","","tax","refund","chart40111","",""
-"sale_tax_exp","0% EXP","","EXP","9995","S","0.0","sale","1","1","tax_group_exp","","base","invoice","","0","0% EXP"
-"","","","","","","","","","","","","tax","invoice","chart40111","0",""
-"","","","","","","","","","","","","base","refund","","0",""
-"","","","","","","","","","","","","tax","refund","chart40111","0",""
+"id","name","description","invoice_label","l10n_pe_edi_tax_code","l10n_pe_edi_unece_category","amount","amount_type","children_tax_ids","type_tax_use","sequence","include_base_amount","tax_group_id","price_include","repartition_line_ids/repartition_type","repartition_line_ids/document_type","repartition_line_ids/account_id","repartition_line_ids/factor_percent","description@es"
```

Comment:
Not sure why we have a total diff here instead of only the added taxes...

Reply:
The `amount_type` and `children_tax_ids` column had to be added for the group tax.
I (quickly) checked that nothing changed for the existing taxes in the old columns with `csvtool`. And the new column values seemed to be fine too.